src/App.css

```diff
@@ -7,3 +7,10 @@
 .game-status-msg {
   margin-top: 0;
 }
+
+.game-change-settings-link{
```

Comment:
Add space between class name to curly brace

Reply:
Please short the CSS declarations according to [this style guide](http://codeguide.co/#css-declaration-order)